src/main/resources/sql/query.sql

```diff
@@ -0,0 +1,16 @@
+CREATE DATABASE yam;
```

Comment:
Actually you don't need explicit ddl's with schema, since hibernate can generate it automatically for you